jsHelper/spicetifyWrapper.js

```diff
@@ -1501,23 +1503,40 @@ Spicetify.Playbar = (function() {
                     #spicetify-update ol > li {
                         list-style-type: decimal;
                     }
+                    .spicetify-update-space {
+                        margin-bottom: 25px;
+                    }
                 </style>
+                <p>Update Spicetify to receive new features and bug fixes.</p>
+                <div class="spicetify-update-space"></div>
                 <p> Current version: ${version} </p>
                 <p> Latest version:
                     <a href="${html_url}" target="_blank" rel="noopener noreferrer">
                         ${semver}
                     </a>
                 </p>
                 <hr>
-                <p>Update Spicetify to receive new features and bug fixes.</p>
-                <ol>
-                    <li>Update Spicetify CLI</li>
+                <div class="spicetify-update-space"></div>
+                <h3>What's Changed</h3>
+                <details>
+                    <summary>
+                        See change logs
+                    </summary>
                     <ul>
-                        <li>Run this command in the terminal:</li>
-                        <pre>spicetify upgrade</pre>
-                        <li> If you installed Spicetify via a package manager, update using said package manager. </li>
+                        ${changelogs()}
                     </ul>
-                    <li>Apply latest changes to Spotify</li>
+                </details>
+                <hr>
+                <div class="spicetify-update-space"></div>
+                <h3>How to update Spicetify?</h3>
```

Comment:
```suggestion
                <h3>Guide</h3>
```

Text must be concise and formal.